src/libs/ReportUtils.ts

```diff
@@ -3649,6 +3693,15 @@ function buildOptimisticChatReport(
     parentReportID = '',
     description = '',
 ): OptimisticChatReport {
+    const participants = participantList.reduce((reportParticipants: Participants, accountID: number) => {
+        const participant: ReportParticipant = {
+            hidden: false,
```

Comment:
This is causing issue https://github.com/Expensify/App/issues/41832, for the transaction threads this needs to be true to be consistent with BE.